test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -1507,6 +1503,12 @@ public void testGetDocContentSCVerifyGroupPush() throws Exception {
       tmp.put(SITES_SITECOLLECTION_VISITORS, Arrays.<Principal>asList());
       goldenGroups2 = Collections.unmodifiableMap(tmp);
     }
+    ContentExchange response1 = new ContentExchange(
```

Comment:
There's a blank line before this in the other test. Let's add one here, too.

Reply:
done